src/EventHub/EventHub/Cmdlets/EventHub/GetAzureEventHubs.cs

```diff
@@ -25,34 +27,45 @@ namespace Microsoft.Azure.Commands.EventHub.Commands.EventHub
     /// <para> If EventHub name provided, a single EventHub detials will be returned</para>
     /// <para> If EventHub name not provided, list of EventHub will be returned</para>
     /// </summary>
-    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "EventHub"), OutputType(typeof(PSEventHubAttributes))]
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "EventHub", DefaultParameterSetName = EventhubPropertiesParameterSet), OutputType(typeof(PSEventHubAttributes))]
     public class GetAzureRmEventHub : AzureEventHubsCmdletBase
     {
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, Position = 0, HelpMessage = "Resource Group Name")]
+        [Parameter(Mandatory = true, Position = 0, HelpMessage = "Resource Group Name", ParameterSetName= EventhubPropertiesParameterSet)]
```

Comment:
Lost the definition for parameter set `NamespaceInputObjectParameterSet`.

Reply:
I introduced NamespaceInputObjectParameterSet and EventHubPropertiesParameterSet. Doing so eliminated the _AllParameterSets that was being evaluated originally. The functionality exhibited by the cmdlet earlier was to evaluate _AllParameterSet. This same functionality is now exhibited by EventHubPropertiesParameterSet, thereby not having any breaking changes (although static analyzer considers this to be breaking change due to the loss of _AllParameterSets). 

The introduction of the two parameter sets was to ensure that InputObject was accepted via the pipeline for the cmdlet.